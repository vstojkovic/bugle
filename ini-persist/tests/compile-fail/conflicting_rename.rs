use ini_persist::LoadProperty;

#[derive(LoadProperty)]
struct Foo {
    #[ini(rename = "argle")]
    #[ini(rename = "bargle")]
    bar: u8,
}

fn main() {
}
