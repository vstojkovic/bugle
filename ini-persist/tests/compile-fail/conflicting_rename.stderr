error: duplicate ini attribute `rename`
 --> tests/compile-fail/conflicting_rename.rs:6:11
  |
6 |     #[ini(rename = "bargle")]
  |           ^^^^^^^^^^^^^^^^^
